src/utils/lupaParser.js

```diff
@@ -1,35 +1,94 @@
 import {
-  KOHTEET,
-  LUPA_SECTIONS
+  GENERIC_LUPA_SECTIONS, VST_LUPA_STRUCTURE
 } from "../scenes/VapaaSivistystyo/modules/constants";
 import { parseLocalizedField } from "../modules/helpers";
+import common from "../i18n/definitions/common";
 
 /**
- * Return an object with lupamääräys data categorized into sections as required in Kuja
+ * Return an object with lupamääräys data categorized into sections matching VST template in
+ * kuja-template/lupahistoria/liikunnankoulutuskeskukset/paatos
+ *
+ * Depends on VST_LUPA_STRUCTURE to guide the order and logic of parsing määräys data and
+ * producting formatted section data, based on kohde or koodisto identifier. Uses react-intl
+ * utilities for primary localization and that based on koodisto data.
+ *
+ * @param lupa lupa with määräys and järjestäjä as given by backend
+ * @param intl react-intl utility object
+ * @returns [{heading: string, values: string[]}]
```

Comment:
According to JSDoc the syntax of parameters is the following (e.g):

@param {string} somebody Somebody's name.

I'm not sure if parameter can be written without { }.

Syntax for @returns is (e.g.): @returns {number} Sum of a and b